Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -2411,7 +2419,7 @@ - (void)handleAppState
             return;
         }
         
-        [self ensureRoomListDataReadyWithCompletion:^{
+        void (^finishAppLaunch)(void) = ^{
```

Comment:
`finishAppLaunch` seemed an ok name, but open to other ideas.

Reply:
Ah it's hard to name them :) that's ok to me